examples/asr/jasper_an4.py

```diff
@@ -95,6 +95,7 @@ def create_dags(model_config_file, vocab, args, nf):
         user_epochs_done_callback=process_evaluation_epoch,
         eval_step=args.eval_freq,
         tb_writer=nf.tb_writer,
+        eval_at_start=not args.do_not_eval_at_start,
```

Comment:
I would suggest going even further and option to turn evaluation off at all (during testing we do not check this value at all, am I right?)

Reply:
Actually I think you're right. I should turn this off for Jasper

Reply:
I added eval_freq to our Jenkins CI job which is larger than the number of steps that this test trains for so eval should no longer happen.
I still need the eval modules so I can test WER after training.